### PR TITLE
First set of changes

### PR DESCRIPTION
**Pull request by imaginesoftware20**
Please approve this PR

### CODE REVIEW

File: app/src/main/java/com/dojo/lit/activity/GamesListActivity.kt [hunk 1]
```diff
@@ -12,17 +12,18 @@ import kotlinx.android.synthetic.main.activity_main.*
 
 class GamesListActivity : BaseActivity() {
     private var tv: TextView? = null
-
+    private var gamelist=  ArrayList<String>()
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/activity/GamesListActivity.kt]: format

Comment (mr-dr) [reply app/src/main/java/com/dojo/lit/activity/GamesListActivity.kt]: make camel case

File: app/src/main/java/com/dojo/lit/lit/PlayGameVM.kt [hunk 2]
```diff
@@ -5,6 +5,7 @@ import kotlinx.android.parcel.Parcelize
 
 @Parcelize
 data class PlayGameVM(
+    val gamecode: String,
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/lit/PlayGameVM.kt]: camel case

File: app/src/main/java/com/dojo/lit/lit/fragment/InitGameFragment.kt [hunk 3]
```diff
@@ -133,12 +130,31 @@ class InitGameFragment : BaseFragment() {
...
-        val logsCountTv = layout.findViewById<EditText>(R.id.logs_count_tv)
+        val logsCountTv = layout.findViewById<Spinner>(R.id.logs_count_dropdown)
         val gameIdTv = layout.findViewById<EditText>(R.id.game_code_tv)
         val aliasTv = layout.findViewById<EditText>(R.id.alias_tv)
-        val playerNoTv = layout.findViewById<EditText>(R.id.player_no_tv)
+        val playerNoTv = layout.findViewById<Spinner>(R.id.player_no_tv)
         val errorMessageTv = layout.findViewById<View>(R.id.error_msg_tv)
 
+        val log_count_array = Arrays.asList("1","2","3","4","5")
+        val player_no_array = Arrays.asList("1","2","3","4","5","6")
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/lit/fragment/InitGameFragment.kt]: format & camel case

File: app/src/main/java/com/dojo/lit/lit/fragment/InitGameFragment.kt [hunk 4]
```diff
@@ -149,15 +165,15 @@ class InitGameFragment : BaseFragment() {
...
             gameIdDialogInput = gameIdTv.text.toString()
             aliasDialogInput = aliasTv.text.toString()
-            playerNoDialogInput = playerNoTv.text.toString()
+            playerNoDialogInput = playerNoTv.selectedItem.toString()
             if((DialogTypes.CREATE_ROOM == dialogType && TextUtils.isEmpty(logsDialogInput))
                 || (DialogTypes.JOIN_ROOM == dialogType && TextUtils.isEmpty(gameIdDialogInput))
                 || TextUtils.isEmpty(aliasDialogInput)
                 || TextUtils.isEmpty(playerNoDialogInput)) {
-                errorMessageTv.visibility = View.VISIBLE
+                Utils.makeToastLong("Please fill all the values!")
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/lit/fragment/InitGameFragment.kt]: move to strings.xml

File: app/src/main/java/com/dojo/lit/lit/fragment/PlayGameFragment.kt [hunk 5]
```diff
@@ -134,17 +136,18 @@ class PlayGameFragment : BaseFragment(), IPlayGameView, View.OnClickListener {
...
 
     private fun setupPresenter() {
         mPresenter = PlayGamePresenter(this, arguments)
     }
 
     override fun setData(vm: PlayGameVM) {
-        Log.d("own_score", "fragment- " + vm.yourScore)
-        Log.d("opponent_score", "fragment- " + vm.opponentScore)
         mDroppedSetsTv.text = getDroppedSetsText(vm.droppedSets)
+        mGameCodeTv.text = getResources().getString(R.string.game_code, vm.gamecode.toString())
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/lit/fragment/PlayGameFragment.kt]: to be done once in setupDefaultValues()

File: app/src/main/java/com/dojo/lit/lit/fragment/PlayGameFragment.kt [hunk 6]
```diff
@@ -174,6 +183,7 @@ class PlayGameFragment : BaseFragment(), IPlayGameView, View.OnClickListener {
             updateCardsInHand(vm.yourCards)
         }
 //        mLogsTv.text = getLogsText(vm.logs)
+        Utils.makeToastLong(vm.logsStr)
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/lit/fragment/PlayGameFragment.kt]: should only be shown once

File: app/src/main/java/com/dojo/lit/lit/fragment/PlayGameFragment.kt [hunk 7]
```diff
@@ -302,10 +335,18 @@ class PlayGameFragment : BaseFragment(), IPlayGameView, View.OnClickListener {
...
         val askableSetNames = mPresenter.getAskableSetNames()
         var askableCards: List<String>? = null
 
+        if(playerName != null && oppositeTeamPlayerNames.contains(playerName)) {
+            var selectedNameTop: MutableList<String>
+            selectedNameTop = oppositeTeamPlayerNames as ArrayList<String>
+            selectedNameTop.remove(playerName)
+            selectedNameTop.add(0,playerName)
+            oppositeTeamPlayerNames = selectedNameTop
+        }
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/lit/fragment/PlayGameFragment.kt]: format

File: app/src/main/java/com/dojo/lit/view/DraggableTextView.kt [hunk 8]
```diff
@@ -18,7 +18,7 @@ class DraggableTextView(context: Context?, attrs: AttributeSet?, val txt: String
     override fun setup() {
         text = txt
         apply { // move to Draggable interface
-            setOnLongClickListener {
+            setOnClickListener {
```
Comment (mr-dr) [inline app/src/main/java/com/dojo/lit/view/DraggableTextView.kt]: to be fixed

**Review by mr-dr — CHANGES_REQUESTED** [review]
card image names are different

File: app/src/main/res/layout/fragment_play_lit.xml [hunk 9]
```diff
@@ -257,12 +268,22 @@
...
             android:layout_height="wrap_content"
-            android:layout_margin="@dimen/standard_margin_half">
+            android:layout_margin="@dimen/standard_margin_half"
+            android:clipToPadding="false"
+            android:layout_gravity="fill_horizontal">
             <ImageView
                 android:layout_width="65dp"
                 android:layout_height="100dp"
                 android:src="@drawable/clubs_2"
+                android:layout_marginRight="-50dp"
```
Comment (mr-dr) [inline app/src/main/res/layout/fragment_play_lit.xml]: remove

File: app/src/main/res/layout/fragment_play_lit.xml [hunk 10]
```diff
@@ -257,12 +268,22 @@
             android:orientation="horizontal"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_margin="@dimen/standard_margin_half">
+            android:layout_margin="@dimen/standard_margin_half"
+            android:clipToPadding="false"
+            android:layout_gravity="fill_horizontal">
```
Comment (mr-dr) [inline app/src/main/res/layout/fragment_play_lit.xml:307]: fix joker getting cut

File: app/src/main/res/layout/init_lit_dialog.xml [hunk 11]
```diff
@@ -7,28 +7,29 @@
...
-    <EditText
-        android:id="@+id/logs_count_tv"
+    <Spinner
+        android:id="@+id/logs_count_dropdown"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:inputType="number"
-        android:hint="Number of logs"
+        android:background="@android:drawable/btn_dropdown"
+        android:hint="@string/number_of_logs"
```
Comment (mr-dr) [inline app/src/main/res/layout/init_lit_dialog.xml:28]: fix hint for dropdown

**Review by mr-dr — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/dojo/lit/activity/GamesListActivity.kt (hunk 1 above)
Comment (imaginesoftware20) [reply app/src/main/java/com/dojo/lit/activity/GamesListActivity.kt]: iska kya matlab?

Comment (imaginesoftware20) [reply app/src/main/java/com/dojo/lit/activity/GamesListActivity.kt]: camel case ka?

**Pull request merged**